### DISCUSSION TITLE
_recid requires inline editing to be on!

### DISCUSSION THREAD

**Discussion by djbbiz** [cortexrd/Knack-Toolkit-Library#160] [General]
It should go without saying, but I made the mistake of using _recid without turning on inline editing.  Attempting to update your Record ID field without this, causes a popup error message to appear when the Grid is rendered that says: 
Record IDs update error: Invalid request

The field can't populate unless you turn on inline editing. The instructions don't mention it as it should be self-evident but I forgot to do it. So if you have this issue turning on inline editing will solve it.

**Comment by cortexrd** [comment]
Oh, yes!  You're absolutely right.  I will update the documentation and add defensive coding to prevent the error.  However, I will write a console log to indicate the discrepancy.

Thanks for reporting,
Norm

**Comment by cortexrd** [comment]
Documentation has been updated.
Error log is sent to console when grid doesn't have **Inline edit** enabled.

Norm
